workspaces/orchestrator/app-config.yaml

```diff
@@ -43,20 +43,23 @@ integrations: {}
 #  github:
 #    - host: github.com
 #      token: ${GITHUB_TOKEN}
-#  gitlab:
-#    - host: gitlab.com
-#      token: ${GITLAB_TOKEN}
 
 proxy:
-  '/quay/api':
-    target: 'https://quay.io'
-    headers:
-      X-Requested-With: 'XMLHttpRequest'
-      # Uncomment the following line to access a private Quay Repository using a token
-      # Authorization: 'Bearer <YOUR TOKEN>'
-    changeOrigin: true
-    # Change to "false" in case of using self hosted quay instance with a self-signed certificate
-    secure: true
+  reviveConsumedRequestBodies: true
+  endpoints:
+    '/quay/api':
+      target: 'https://quay.io'
+      headers:
+        X-Requested-With: 'XMLHttpRequest'
+        # Uncomment the following line to access a private Quay Repository using a token
+        # Authorization: 'Bearer <YOUR TOKEN>'
+      changeOrigin: true
+      # Change to "false" in case of using self hosted quay instance with a self-signed certificate
+      secure: true
+    '/mytesthttpserver':
```

Comment:
since this is for dev only, is ok to have this endpoint in the app-config? Not sure if a `app-config-dev` would fit better (if possible)

Reply:
I think it's not an issue. This Backstage is used for development only. It is never distributed and deployed in production.